composer.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "datacodetech/phpcs-ruleset",
-	"version": "2.2.0",
+	"version": "2.3",
```

Comment:
`2.2.1` would be fine for this change. Even thought code that was previously okay might hist this rule now It's more a clarification than a deliberate change.